index.html

```diff
@@ -45,6 +45,224 @@ <h1 class="name">I’m Ali Hassan. Glad to see you!</h1>
           <div class="connect">
             <p>LET'S CONNECT</p>
           </div>
+          <div>
+            <ul class="social">
+              <li>
+                <a class="twitter" href="#"
+                  ><img src="./images/main/Vector-3.png" alt="twitter"
+                /></a>
+              </li>
+              <li>
+                <a class="linkedin" href="#"
+                  ><img src="./images/main/Linkedin icon.svg" alt="linkedin"
+                /></a>
+              </li>
+              <li>
+                <a class="M" href="#"
+                  ><img src="./images/main/M.svg" alt="M"
+                /></a>
+              </li>
+              <li>
+                <a class="github" href="#"
+                  ><img src="./images/main/Lomri.svg" alt="github"
+                /></a>
+              </li>
+              <li>
+                <a class="V" href="#"
+                  ><img src="./images/main/V.svg" alt="V"
+                /></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </section>
+      <section id="works-section">
+        <div class="work-projects">
+          <div class="item project-header">
+            <div class="project-img-container">
+              <img src="./images/work/img_1.PNG" alt="" />
+            </div>
+          </div>
+          <div class="card-content">
+            <div class="item project-description">
+              <div class="project-title-container">
+                <h4 class="description-title">Tonic</h4>
+              </div>
+              <div class="project-subtitle-container">
+                <ul class="subtitle-items">
+                  <li class="list-less"><p>CANOPY</p></li>
+                  <li class="list"><p>Back End Dev</p></li>
+                  <li class="list"><p>2015</p></li>
+                </ul>
+              </div>
+              <div class="project-article-container">
+                <p class="description-item description-item-b">
+                  A daily selection of privately perosonalized reads; no
+                  accounts or sign-ups required.
+                </p>
+              </div>
+              <div class="project-tag-container">
+                <ul class="tag-item">
+                  <li class="tags"><a class="tag-btn" href="#">html</a></li>
+                  <li class="tags"><a class="tag-btn" href="#">css</a></li>
+                  <li class="tags">
+                    <a class="tag-btn" href="#">javaScript</a>
+                  </li>
+                </ul>
+              </div>
+              <div class="project-button-container">
+                <a href="#" class="project-button">See project</a>
```

Comment:
Nice job adding button interaction styling on hover✨. However, please also add styling for when the button is pressed using the `:active` pseudoselector using the figma style guide. Please do this for **all** your buttons

<img width="351" alt="image" src="https://github.com/mhassa204/Portfolio-Project/assets/37116322/8b7e0665-2e6e-48f6-8348-7060b16c53b0">